keyboards/reviung/reviung46/keyboard.json

```
@@ -0,0 +1,113 @@
+{
+    "keyboard_name": "reviung46",
+    "manufacturer": "gtips",
+    "url": "https://github.com/gtips/reviung/tree/master/reviung46/",
+    "maintainer": "gtips",
+    "usb": {
+        "vid": "0x4E95",
+        "pid": "0x4E19",
+        "device_version": "0.0.1"
+    },
+    "rgblight": {
+        "hue_steps": 16,
+        "saturation_steps": 16,
+        "brightness_steps": 16,
+        "led_count": 10,
+        "sleep": true,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "D3"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false,
```

Generally preferred to have these enabled if you're able to.
```suggestion
        "mousekey": true,
        "nkro": true,
```

---

Thanks.
I fixed it.